test/integration/index.js

```diff
@@ -38,7 +38,7 @@ beforeEach(() => {
 
   // Clear all data out of the test database
   return Promise.all([
-    models.sequelize.truncate({ cascade: true }),
+    models.sequelize.truncate({ cascade: true, restartIdentity: true }),
```

Comment:
I'm not sure how our tests ever worked well without this (especially the cache key snapshots), but either way this should help make our test suite a bit more deterministic.

cc @gimenete

Reply:
Hmmm, cache keys have always been based on Slack and GitHub ids as far as I understand. Those ids are written by hand in tests and fixtures. That's why they've worked.

I see that you are now using the `workspaceId`. Can we get an id from Slack for this so we make our tests not dependant on ids based on sequences?

Reply:
> Hmmm, cache keys have always been based on Slack and GitHub ids as far as I understand.

From what I can tell most cache keys that cache the workspace id use our database id. I included cache key examples in my report here: https://github.com/github/ecosystem-primitives/pull/229/files?short_path=0d9829a#diff-0d9829a4a83bd19741db3959bf362d99

I believe the logic for all of those is this method:
https://github.com/integrations/slack/blob/cb787ec09865b518410fc1f4e9d2e34466018667/lib/models/subscription.js#L104-L106

Reply:
Hmmm ok.

At some point in the future I think we should make `SlackWorkspaces.id` the real Slack id and here `slackWorkspaceId` would be also the Slack id, and not an internal id.

Reply:
yeah i think that makes sense, though I don't look forward to that migration haha 👍

Reply:
hahaha, yeah, it's a tough one 😜